download-kafka.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh -e
+KAFKA_VERSION=1.1.0
```

Comment:
KAFKA_VERSION and SCALA_VERSION are provided by the Dockerfile so shouldn’t be added here.